### ISSUE TITLE
Misunderstood self connection?

### ISSUE THREAD

**Issue by PeterKW** [pyamsoft/tetherfi#198]
I've been looking for a way of turning on wireless debugging while outside of normal # WiFi connection spaces. So that I could activate Shizuku without WiFi.

I think that perhaps this project doesn't help with what I was trying as the device running the tetherfi networking can't itself connect to the network even though WiFi isn't turned off when this 'hotspot' is formed on?

**Comment by pyamsoft** [comment]
Yeah, the hosting device cannot connect to the wifi direct network. A root app may be able to access hostapd directly, but at that point you wouldn't need Shizuku either
